libtiledbvcf/src/stats/column_buffer.cc

```diff
@@ -178,7 +178,7 @@ size_t ColumnBuffer::update_size(const Query& query) {
     if (offsets_.capacity() < num_offsets + 1) {
       offsets_.reserve(num_offsets + 1);
     }
-    offsets_[num_offsets] = num_elements;
+    *(offsets_.data() + num_offsets) = num_elements;
```

Comment:
Why?

Reply:
MSVC had an issue with accessing offsets_ vector out of bounds, which it actually does since it uses `capacity/reserve` instead of `size/resize`. I tried to change it in order to remove this bound check. I was getting array access out of bounds.